frontend/src/store/modules/projects.js

```diff
@@ -47,9 +47,9 @@ const actions = {
   },
   async delete ({ commit, state }, { metadata }) {
     const namespace = metadata.namespace
-    await deleteProject({ namespace })
-    commit('ITEM_DELETED', metadata)
-    return state.all
+    const res = await deleteProject({ namespace })
+    // commit('ITEM_DELETED', metadata)
```

Comment:
Why is this line commented out. If we don't need it anymore it should be removed. Do we really need the return value of this action?

Reply:
write a comment why we do not delete from store.

Reply:
delete mutation